requirements.txt

```diff
@@ -1,2 +1,29 @@
-pandas>=1.5.0
-scikit-learn>=1.2.0
+#
```

Comment:
Q: do we need `requirements.txt` in the repo at all now? If you delete `setup.py` and move all those details to `pyproject.toml`, then I don't think that requirements has a purpose any more (other than perhaps listing indirect dependencies and where they come from).

Reply:
listing the indirect dependencies was the intent but ultimately this will be different based on the environment tubular is being installed into so probably not that useful.  Will delete.

Reply:
https://github.com/lvgig/tubular/pull/271/commits/fad971fc3c94fa2c20e4c60c3e3757dda7f37ed9